qiskit_rigetti/_qcs_backend.py

```diff
@@ -143,6 +143,10 @@ def run(
         if not isinstance(run_input, list):
             run_input = [run_input]
 
+        bindings = options.get("parameter_binds") or []
+        if len(bindings) > 0:
```

Comment:
```suggestion
        if bindings:
```
Would cover falsey cases without creating a new list if the change above is rejected in case of `None`.

Reply:
I'll stick with being explicit here, so it's easier to think of `bindings` as an array always